### DISCUSSION TITLE
Feedback if CalculationAll is done?

### DISCUSSION THREAD

**Discussion by afunrwth** [Dlubal-Software/RSTAB_Python_Client#8] [Q&A] [answered]
Is there a way to get feedback wether CalculationAll is done - to start further steps?

**Comment by OndraMichal** [comment, marked as answer]
Hi @afunrwth,
using SOAP api all commands runs in a sequence. Feedback can be provided by any command put behind Calculate_All() like print(....). Calculate_all() also returns calculationMessages.
